frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -530,6 +544,13 @@ export const Chart = ({
         </div>
         {!isMobile && <Filters {...chartFiltersProps} />}
       </header>
+      {!chartDataMissing && hasChartAnimationParam &&
+          <div
+            style={{ minHeight: chartHeight }}
+            className={`
+          ${styles.transitionDiv}
+          ${showAnimationOverlay ? '' : styles.overlayRemove}`}
+          />}
```

Comment:
nit: indentation looks a bit funny, I'd always wrap JSX like this in `{conditions && ( \newline)`